serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc

```diff
@@ -50,69 +59,87 @@ Kogito follows the link:{spec_doc_url}#using-functions-for-restful-service-invoc
 
 [NOTE]
 ====
-The function `type` attribute can be ommited since the link:{spec_doc_url}#Function-Definition[default value] is `rest`.
+In the previous example function definition, the `type` attribute can be ommited as the link:{spec_doc_url}#Function-Definition[default value] is `rest`.
 ====
 
-The `operation` attribute is a string composed by:
+In the previous example, the `operation` attribute is a string, which is composed using the following parameters:
 
-1. The URI that the engine uses to locate the specification file
-2. The operation identifier (`operationId`), which you find in the link:{open_api_spec_url}#fixed-fields-7[OpenAPI specification file].
+* URI that the engine uses to locate the specification file, such as `classpath`.
+* Operation identifier. You can find the operation identifier in the link:{open_api_spec_url}#fixed-fields-7[OpenAPI specification file].
 
 Kogito supports the following URI schemes:
 
-* `classpath` for files located in the application project under the `src/main/resources` folder. The `classpath` is the default scheme when you don't define a scheme in the string. For example `classpath:/myopenapifile.yaml` or `/myopenapifile.yaml`. In this example, the file location is in `src/main/resources/myopenapifile.yaml` path
-* `file` for files located in the filesystem
-* `http` or `https` for remotely located files
+* `classpath`: This URI scheme is supported for the files that are located in the `src/main/resources` folder of the application project. The `classpath` is the default URI scheme. If you do not define a URI scheme as shown in the following examples, the file location is defined as `src/main/resources/myopenapifile.yaml`:
++
+--
+`classpath:/myopenapifile.yaml`
+
+`/myopenapifile.yaml`
+-- 
+
+* `file`: This URI scheme is supported for the files that are located in the file system.
+* `http` or `https`: This URI scheme is supported for remotely located files.
 
 [IMPORTANT]
 ====
-The OpenAPI specification files must be available only on build time. This is because, during runtime, Kogito leverages its internal code generation feature to make the REST calls. So {build-workflow-image-with-quarkus-cli}[once you build your application image], the Kogito engine won't access the files anymore.
-====
-
-== Getting Access to the OpenAPI Specification Files
+The OpenAPI specification files must be available during the build time, because Kogito leverages the internal code generation feature to send requests on runtime. Therefore, when you build your application image, the engine will not have the access to the files. 
```

Comment:
I think we need to review this part. I feel like the sentence "When you build your application image, the engine will not have the access to the files." lost its meaning in the review. The application needs access to the file only when building the application or the image. So "once you have built the image, Kogito won't access the files anymore".